elastic/kibana
# [Synthetics] adjust monitor status rule location logic

## Summary

Resolves https://github.com/elastic/kibana/issues/156223

In previous stack versions, the label was not saved to the saved object. 

This ensures that alerting works both for those early versions as well current versions, by adjusting the logic to consider both the location.label and location.id in alerting logic.

### Testing

1. Create a new oblt-cli cluster and connect to the synthetics service, locally or via the dev environment
2. Checkout v8.5.3
3. Create a monitor. Make sure it's down and wait for the first down check
4. Check out this branch
5. Navigate to Observabillity alerts. Ensure that an alert is generated for the monitor. You may have to wait a few minutes.
